packages/contracts-bedrock/scripts/DeployConfig.s.sol

```diff
@@ -173,4 +174,19 @@ contract DeployConfig is Script {
         bytes memory res = vm.ffi(cmd);
         return abi.decode(res, (bytes32));
     }
+
+    function getDevAccounts() public pure returns (address[10] memory) {
```

Comment:
Any reason this is in the deploy config and not the deploy helpers?

Reply:
I thought it might be useful in the future for other contracts that need to know the dev accounts if `fundDevAccounts` is enabled in the deploy config. I can move this to the helpers contracts if we'd rather have it there until another contracts needs access to the dev account addresses

Reply:
I think this goes in the helpers much more cleanly, its not necessarily config, its a constant